doc/can-observe.md

```diff
@@ -309,6 +309,54 @@ class WidgetViewModel {
 }
 ```
 
+## Extending can-observe with Observables
+
+can-observe recognizes a `can.computedPropertyDefinitions` property; getting or setting a key on the object that matches a key in the `can.computedPropertyDefinitions` object will use this functionality. Specifically, the values in this object are functions which return an observation (typically with [can-observation]). The first time one of these properties is accessed, the function is run, and the observation is cached, to be used for all future use on _that instance_.
```

Comment:
>  getting or setting a key on the object

Which object?  The `proxy-wrapped` object.  I think I might call them proxy-wrapped elsewhere, if not, you might want to establish this term.

Reply:
is `can.computedPropertyDefinitions` formally described somewhere?  We might want to do that.

Reply:
Where would this formal definition go? in can-observe? in can-symbols?

Reply:
in can-observe